source/gems/dry-component/booting-a-dependency.html.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Booting a Dependency
 layout: gem-single
+name: dry-component
```

Comment:
@solnic Now I have realised that it's getting a bit convoluted with pages that 2 level deep nesting (i.e. dry-validation > Basics > Predicate Logic)